RunningOrder/Sprint/NewSprintView.swift

```diff
@@ -31,9 +34,11 @@ struct NewSprintView: View {
     }
 
     private func createSprint() {
-        let sprint = Sprint(name: name, stories: [])
-        self.createdSprint = sprint
-        dismiss()
+        if number != nil && name != "" {
+            let sprint = Sprint(number: number!, name: name, color: .init(0x2FBAD5), stories: [])
```

Comment:
if let

Reply:
why create a intermediate variable when you set self.createdSprint the line after ? :p

Reply:
ah ! that was my code before, but not catch in the MR :p